dockerspawner/dockerspawner.py

```diff
@@ -965,7 +883,9 @@ async def get_command(self):
         else:
             image_info = await self.docker("inspect_image", self.image)
             cmd = image_info["Config"]["Cmd"]
-        return cmd + self.get_args()
+            args = self.get_args()
+        return cmd #+ ['--auth=none']
+        #+ self.get_args()
```

Comment:
Is this change intentional? You get the arguments but you do not do anything with them. Also, please remove commented out code

Reply:
Why are the `args` no longer necessary? Does the super implementation provide some that are unsuitable?

Reply:
The implementation adds flags that are not supported by code-server. They added those for the communication between the Hub and the Notebooks. We already have all the flags and arguments needed for code-server in our dockerfile, from where this implementation also gets the command.